qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -350,20 +350,31 @@ def convert_play(self, shift, instruction):
         Returns:
             dict: Dictionary of required parameters.
         """
+        start_time = shift + instruction.start_time
+        channel = instruction.channel.name
+
         if isinstance(instruction.pulse, library.ParametricPulse):
+            if instruction.pulse.name:
+                pulse_name = instruction.pulse.name
+            else:
+                pulse_name = str(hash(tuple(instruction.pulse.parameters.values())))
```

Comment:
These names are going to be senselessly long. The hash also does not include the type of parametric pulse.